utility_analysis_new/dpengine.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2022 OpenMined.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""DPEngine and ContributionBounder for utility analysis."""
+import collections
+
+import pipeline_dp
+from pipeline_dp import DPEngine
```

Comment:
Nit: by Google Python coding style modules should be imported not classes function. Please use `pipeline_dp.DPEngin`e in code. 

It's possible to use aliases for modules, like
from <module> import <sub-module> as <short name>

Reply:
This applies to other classes as well, right? E.g. ContributionBounder or DataExtractor?

Reply:
Yes